payload/builder.go

```diff
@@ -81,6 +96,42 @@ func (p *Payload) Alert(alert interface{}) *Payload {
 	return p
 }
 
+// SetContentState sets the aps content-state on the payload.
+// This will update content-state of live activity widget.
+//
+//	{"aps":{"content-state": {} }}`
+func (p *Payload) SetContentState(contentState map[string]interface{}) *Payload {
+	p.aps().ContentState = contentState
+	return p
+}
+
+// SetDismissalDate sets the aps dismissal-date on the payload.
+// This will end live activity on dismissal date timestamp.
+//
+//	{"aps":{"dismissal-date": DismissalDate }}`
+func (p *Payload) SetDismissalDate(dismissalDate float64) *Payload {
+	p.aps().DismissalDate = dismissalDate
+	return p
+}
+
+// SetDismissalDate sets the aps dismissal-date on the payload.
+// This will end live activity on dismissal date timestamp.
+//
+//	{"aps":{"dismissal-date": DismissalDate }}`
+func (p *Payload) SetEvent(event ELiveActivityEvent) *Payload {
+	p.aps().Event = event
+	return p
+}
```

Comment:
I think the doc string is still wrong here, something like this maybe?

```suggestion
// SetEvent sets the aps event type on the payload.
// This can either be `LiveActivityEventUpdate` or `LiveActivityEventEnd`
//
//	{"aps":{"event": Event }}`
func (p *Payload) SetEvent(event ELiveActivityEvent) *Payload {
	p.aps().Event = event
	return p
}
```